src/coreComponents/fieldSpecification/TractionBoundaryCondition.cpp

```diff
@@ -220,6 +234,33 @@ void TractionBoundaryCondition::launch( real64 const time,
   }
 }
 
+void TractionBoundaryCondition::reinitScaleSet( FaceManager const & faceManager,
```

Comment:
I am not sure I understand what's happening in here and why you need it.

Reply:
> I am not sure I understand what's happening in here and why you need it.

I added this function to account for the reduction of traction on the boundary where phase-field damage is generated. Maybe this function can be used for other purposes when the traction magnitude depends on any nodal variables.